tests/Unit/CanonicalJsonTraitTest.php

```diff
@@ -24,6 +24,28 @@ public function testSort(): void
         $this->assertSame($unsortedData, $sortedData);
     }
 
+    /**
+     * Indexed arrays with >= 10 items should not be changed.
+     */
+    public function testIndexedArraysAreNotChanged(): void
+    {
+        $original = [
+          'b' => array_fill(0, 20, 'Hello!'),
+          'a' => 'Canonically speaking, I go before b.',
+        ];
```

Comment:
Isn't the other thing about array_is_list is the numeric keys have to be consecutive?

should we test with this also?

$array = [
]

Reply:
Yeah, the array does need to have consecutive integer keys, or it's not considered a list.